change/@ni-nimble-components-c6c83088-a25b-49f2-bd8f-50abc517ede5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "created Computations module",
+  "packageName": "@ni/nimble-components",
+  "email": "natan.muntean@ni.com",
```

Comment:
In my PR I got this comment, maybe will help you too :) 
"In [the root CONTRIBUTING.md](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#beachball-change-file) there are instructions about how to configure GitHub to use a private email address to avoid exposing your NI address in public. For this PR you can manually modify this JSON file to copy in the address that GitHub shows at https://github.com/settings/emails"

https://github.com/ni/nimble/pull/818#discussion_r1021708842

Reply:
thanks!